R/style_scale_color.R

```diff
@@ -67,40 +95,21 @@ cllr_palettes <- list(
 
 #################################
 
-#' Grab any combination of colors for our palettes
+#' Access Palettes
+#'
+#' @description Access any of the color palettes available
+#'
+#' @param palette Character name of palette in cllr_palettes
+#' @param reverse Boolean; Should the colors be reversed?
 #'
-#' @param ... colors ya want
 
-cllr_colors <- function(...){
-  cols <- c(...)
-  if (is.null(cols))
-    return(colors)
 
-  colors[cols]
+cllr_pal <- function(palette = "main", reverse=F, ...){
+  pal <- cllr_palettes[[palette]]
```

Comment:
In the documentation at the top I got the impression that the palette names woere uppercase and now it looks like they are lower?

If you want it to handle upper and lower you could do something like
```r
cllr_palettes[[grepl(palette, names(cllr_palettes), ignore.case = T)]]
```

Although, because they are stored in a list you can get back multiple palettes, which could cause issues. A switch statement or a defensive check would prevent returning multiple palettes